src/style.css

```diff
@@ -5,7 +5,7 @@
 }
 
 body {
-  font-family: 'poppins', sans-serif;
+  font-family: "poppins", sans-serif;
```

Comment:
- [ ] Your app working well 👌🏼👏🏼🙌🏼. Please kindly rerecord your video presentation 📽 because it should be max 5 min long, and yours is 10 min 👍🏼

Reply:
- [ ] [OPTIONAL] You have submitted a project, The pull request link has not opened correctly. In fact for the group projects while you are working, we were supposed to give reviews to each other and merge their feature branches into the dev branch then open a pull request from the dev branch into the main branch please next time for your capstone group project follow  [this article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/code_review_flow_group_projects.md) .